lib/credentials/remote_credentials.js

```diff
@@ -217,3 +234,15 @@ AWS.ECSCredentials = AWS.util.inherit(AWS.Credentials, {
     });
   }
 });
+
+AWS.ECSCredentials = AWS.util.inherit(AWS.RemoteCredentials, {
```

Comment:
`AWS.ECSCredentials` should be in `lib/credentials/ecs_credentials.js` (the file path previously used), and that file should require `AWS.RemoteCredentials`. Some customers may be directly importing ECS Credentials as a submodule, so its location on disk is a public interface.